build_stage_repository

```diff
@@ -120,11 +120,21 @@ def create_modulemd(collection, version, stage_dir, foreman_version, dist):
     with open(modules_yaml, 'w') as modules_file:
         yaml.dump_all(modules, modules_file, default_flow_style=False, explicit_start=True, explicit_end=True)
 
-    check_output(['modifyrepo_c', '--compress-type=gz', '--mdtype=modules', modules_yaml, f"{stage_dir}/repodata"])
+    try:
+        check_output(['modifyrepo_c', '--compress-type=gz', '--mdtype=modules', modules_yaml, f"{stage_dir}/repodata"])
```

Comment:
Should we combine stdout & stderr into one stream to maintain order?

Reply:
that's also a possibility.